pandas/core/window.py

```diff
@@ -321,16 +321,21 @@ def validate(self):
         if isinstance(window, (list, tuple, np.ndarray)):
             pass
         elif com.is_integer(window):
-            try:
-                import scipy.signal as sig
-            except ImportError:
-                raise ImportError('Please install scipy to generate window '
-                                  'weight')
-
-            if not isinstance(self.win_type, compat.string_types):
-                raise ValueError('Invalid win_type {0}'.format(self.win_type))
-            if getattr(sig, self.win_type, None) is None:
-                raise ValueError('Invalid win_type {0}'.format(self.win_type))
+            if window < 0:
```

Comment:
actually, maybe this is not needed here. let scipy handle negative weights (if it can) or bork if it can't. Pls add a couple of tests and see what happens.